easy-rules-core/src/main/java/org/jeasy/rules/core/DefaultRulesEngine.java

```diff
@@ -149,6 +150,12 @@ private void log(Rules rules) {
 
     private void log(Facts facts) {
         LOGGER.debug("Known facts:");
+
+        if (Objects.isNull(facts)){
```

Comment:
We should not accept `null` in the first place. Please update the PR and add `Objects.requireNonNull` for the `Rules` and `Facts` parameters in `fire` and `check` methods.